Sitemap/Url/UrlConcrete.php

```diff
@@ -173,11 +209,10 @@ public function toXml()
     }
 
     /**
-     * basic url has no namespace. see decorated urls
-     * @return array
+     * @inheritdoc
      */
     public function getCustomNamespaces()
     {
-        return array();
+        return array(); // basic url has no namespace. see decorated urls
```

Comment:
Some arrays converted to short arrays, some not 😕

Reply:
I tried not to write short syntax arrays, where did you find some in the introduced code ?

We may do an other PR with array short syntax uniformisation